src/auth/passwordreset/redux/validations.js

```diff
@@ -0,0 +1,4 @@
+const loginUtil = {
+  LOGIN_UTILS: /^[a-zA-Z0-9]+@+[a-zA-Z0-9]+.+[A-z]/,
```

Comment:
@HabibSentongo consider renaming this files to something relating to this feature you are working on. Preferably **passwordResetUtil** or else any other.

Reply:
This has been resolved, thank you.